sensor_app/adapters/secondary/background_jobs_celery/background_jobs_repo.py

```diff
@@ -0,0 +1,78 @@
+# adapters.py
+import asyncio
+from celery import Celery
+from sensor_app.core.ports.secondary import BackgroundJobsRepository
+from sensor_app.core.ports.secondary import SensorRepository
+from sensor_app.settings import BackgroundJobsSettings
+from sensor_app.core.use_cases.sensor import MakeOneThousandSensors
+from sensor_app.core.domain.results import AsyncResult
+
+# Create a global singleton so this can be referenced in the repo and in sensor_app.main
+_celery_app = None
+
+def configure_usecases_as_tasks(celery_app, sensor_repo) -> None:
+    @celery_app.task(name='make_one_thousand_sensors')
+    def make_one_thousand_sensors():
+        loop = asyncio.get_event_loop()
+        results = loop.run_until_complete(MakeOneThousandSensors(sensor_repo=sensor_repo)())
+        return AsyncResult(
+            id=results.id,
+            status=results.status,
+            result=str(results.result),
+            traceback=results.traceback,
+            date_done=str(results.date_done)
+        )
```

Comment:
Not particularly happy about this part... how to tag our use cases that we want to run as background jobs...?